code/datums/craft/recipes/clothing.dm

```diff
@@ -263,6 +263,17 @@
 		list(QUALITY_ADHESIVE, 15, 30)
 	)
 
+/datum/craft_recipe/clothing/fullhelm_bs
+	name = "blackshield full helmet"
+	result = /obj/item/clothing/head/helmet/ballistic/shieldfull
+	steps = list(
+		list(/obj/item/clothing/head/helmet/ballistic/militia, 1, "time" = 30),
+		list(CRAFT_MATERIAL, 2, MATERIAL_STEEL),
```

Comment:
Full helmets offer a bit more protection than the average helmet. I suggest adding at least 2 sheets of plasteel as well to the recipe to make sense balance-wise.

Reply:
Agree